.github/ISSUE_TEMPLATE.md

```diff
@@ -1,41 +1,41 @@
+<!--
 ## Before you submit an issue
 Please look up our issues, maybe your problem was mentioned before.
+-->
 
 ## Description
-Provide a short description of your problem here.
+<!-- Provide a short description of your problem here. -->
 
 If you executed a specific command:
 ```shell
-your command here
+# your command here
 ```
 
 Output:
-```shell
-output
-of
-your
-command
-here
+```
+output of your command here
 ```
 
 ## Versions and system information
```

Comment:
Would it make sense to just have a long shell command here that people could paste the output from?
e.g. Please include the output of the following command (Just paste into your terminal and hit enter).
```
echo -n "\nCrew version: `crew --version`\nModel: `cat /etc/lsb-release | grep CHROMEOS_RELEASE_DESCRIPTION | awk '{ print $(NF)}'`\nMilestone: `cat /etc/lsb-release | grep CHROMEOS_RELEASE_CHROME_MILESTONE | awk -F= '{ print $(NF)}'`\nChannel: `cat /etc/lsb-release | grep CHROMEOS_RELEASE_TRACK | awk -F= '{ print $(NF)}'`\nArchitecture: `uname -m`\n\n`cat /etc/lsb-release`\n" 
```

In my container terminal I get the following from that:
```
Crew version: 1.21.3
Model: nocturne
Milestone: 90
Architecture: x86_64

CHROMEOS_ARC_ANDROID_SDK_VERSION=28
CHROMEOS_ARC_VERSION=7334568
CHROMEOS_AUSERVER=https://tools.google.com/service/update2
CHROMEOS_BOARD_APPID={BD7F7139-CC18-49C1-A847-33F155CCBCA8}
CHROMEOS_CANARY_APPID={90F229CE-83E2-4FAF-8479-E368A34938B1}
CHROMEOS_DEVSERVER=
CHROMEOS_RELEASE_APPID={BD7F7139-CC18-49C1-A847-33F155CCBCA8}
CHROMEOS_RELEASE_BOARD=nocturne-signed-mpkeys
CHROMEOS_RELEASE_BRANCH_NUMBER=82
CHROMEOS_RELEASE_BUILDER_PATH=nocturne-release/R90-13816.82.0
CHROMEOS_RELEASE_BUILD_NUMBER=13816
CHROMEOS_RELEASE_BUILD_TYPE=Official Build
CHROMEOS_RELEASE_CHROME_MILESTONE=90
CHROMEOS_RELEASE_DESCRIPTION=13816.82.0 (Official Build) stable-channel nocturne
CHROMEOS_RELEASE_KEYSET=mp
CHROMEOS_RELEASE_NAME=Chrome OS
CHROMEOS_RELEASE_PATCH_NUMBER=0
CHROMEOS_RELEASE_TRACK=stable-channel
CHROMEOS_RELEASE_UNIBUILD=1
CHROMEOS_RELEASE_VERSION=13816.82.0
DEVICETYPE=CHROMEBOOK
GOOGLE_RELEASE=13816.82.0
```

Reply:
How about creating a function for this in `crew`? (it can be `crew sysinfo`)

Reply:
See #6875